modules/core/src/test/java/org/apache/ignite/internal/sql/SqlParserBulkLoadSelfTest.java

```diff
@@ -207,8 +207,25 @@ public void testCopyParseProperties() {
         assertEquals(actual.getClass().getSimpleName(), "UnmodifiableMap");
 
         // quotes required
-        GridTestUtils.assertThrows(log(),
-                () -> new SqlParser(null,"COPY FROM 'a' INTO 'b' FORMAT ICEBERG PROPERTIES (unquotedKey=unqotedVal)").nextCommand(),
-                SqlStrictParseException.class, "Unexpected token: \"=UNQOTEDVAL\" (expected: \"=\")");
+        assertParseError(null,
+                "COPY FROM 'a' INTO 'b' FORMAT ICEBERG PROPERTIES (unquotedKey=unqotedVal)",
+                "Unexpected token: \"=UNQOTEDVAL\" (expected: \"=\")");
+    }
+
+    @Test
+    public void testLocationsOrder() {
+        new SqlParser(null,
+                "copy from 'any.file' into Person (_key, age, firstName, lastName) format csv")
+                .nextCommand();
+        new SqlParser(null,
+                "copy into 'any.file' from Person (_key, age, firstName, lastName) format csv")
+                .nextCommand();
+
+        assertParseError(null,
+                "copy into 'any.file' into Person (_key, age, firstName, lastName) format csv",
+                "(expected both locations: \"FROM\", \"INTO\")");
+        assertParseError(null,
```

Comment:
I don't see reverse order test, like a `copy from Person (_key, age, firstName, lastName) format csv into 'any.file'`, and so one